assets/js/custom/Translation.js

```diff
@@ -2,6 +2,111 @@
 
 // eslint-disable-next-line no-unused-vars
 function Translation (hasDescription, hasCredit) {
+  let languageMap = {}
+  let targetLanguage = document.documentElement.lang
+
+  $(document).ready(function () {
+    $.ajax({
+      url: '../languages',
+      type: 'get',
+      success: function (data) {
+        languageMap = data
+      }
+    })
+  })
+
+  $('.mdc-list-item__text').each(function () {
+    if ($(this).parent()[0].hasAttribute('selected')) {
+      targetLanguage = fixLanguageCode($(this).parent().attr('data-value'))
+    }
+  })
+
+  $(document).on('click', '.comment-translation-button', function () {
+    const commentId = $(this).attr('id').substring('comment-translation-button-'.length)
+
+    $(this).hide()
+
+    if ($('#comment-text-translation-' + commentId).attr('lang') !== targetLanguage) {

```

Comment:
to make this more readable, refactor this expression to a function like `isTranslationAlreadyAvailable` and replace this with `if (isTranslationAlreadyAvailable(...)`

Reply:
Created a new isTranslationAvailable helper function